client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -371,7 +372,7 @@ class CalypsoifyIframe extends Component<
 		}
 
 		if ( EditorActions.GetGutenboardingStatus === action ) {
-			const isGutenboarding = this.props.siteCreationFlow === 'gutenboarding';
+			const isGutenboarding = this.props.siteCreationFlow === 'gutenboarding' && ! this.props.plan;
```

Comment:
This variable is evaluated in ET plugin to decide if the Launch modal should open or the user should be redirected to the classic Launch flow: https://github.com/Automattic/wp-calypso/blob/master/apps/editing-toolkit/editing-toolkit-plugin/editor-gutenboarding-launch/index.ts#L92